src/inference/include/openvino/runtime/core.hpp

```diff
@@ -42,7 +42,7 @@ class OPENVINO_RUNTIME_API Core {
     class Impl;
     std::shared_ptr<Impl> _impl;
 
-    void get_property(const std::string& device_name, const std::string& name, ov::Any& to) const;
+    void get_property(const std::string& device_name, const std::string& name, const AnyMap& arguments, Any& to) const;
```

Comment:
Why do we need this change?

Reply:
Some properties require additional arguments such as MODEL_PTR for MAX_BATCH_SIZE, so that is why we need an additional map. This has already been fixed in separate PR